{"uuid":"b5b357f8-c676-4c53-81ac-15e3fa6f8f53","name":"Scenario Outline: The user getting error message with invalid python code for Types of Linked List","children":["f8165416-4a47-41a2-b788-a565e64d5c41"],"befores":[],"afters":[{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696389816356,"stop":1696389816356},{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696389816506,"stop":1696389816506},{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696389816573,"stop":1696389816573},{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696389818634,"stop":1696389818635}],"start":1696389815950,"stop":1696389818640}